
These timestamp warnings are well-intended but totally unhelpful in reality.

diff -rup a/fs/namespace.c b/fs/namespace.c
--- a/fs/namespace.c	2024-12-09 10:41:16.000000000 +0100
+++ b/fs/namespace.c	2024-12-14 16:45:32.000899975 +0100
@@ -2941,6 +2941,7 @@ static void set_mount_attributes(struct
 	touch_mnt_namespace(mnt->mnt_ns);
 }
 
+__attribute__((unused))
 static void mnt_warn_timestamp_expiry(struct path *mountpoint, struct vfsmount *mnt)
 {
 	struct super_block *sb = mnt->mnt_sb;
@@ -3002,8 +3003,6 @@ static int do_reconfigure_mnt(struct pat
 	unlock_mount_hash();
 	up_read(&sb->s_umount);
 
-	mnt_warn_timestamp_expiry(path, &mnt->mnt);
-
 	return ret;
 }
 
@@ -3054,8 +3053,6 @@ static int do_remount(struct path *path,
 		up_write(&sb->s_umount);
 	}
 
-	mnt_warn_timestamp_expiry(path, &mnt->mnt);
-
 	put_fs_context(fc);
 	return err;
 }
@@ -3439,8 +3436,6 @@ static int do_new_mount_fc(struct fs_con
 	if (IS_ERR(mnt))
 		return PTR_ERR(mnt);
 
-	mnt_warn_timestamp_expiry(mountpoint, mnt);
-
 	mp = lock_mount(mountpoint);
 	if (IS_ERR(mp)) {
 		mntput(mnt);
